ackermann_steering_controller/CMakeLists.txt

```diff
@@ -0,0 +1,107 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(ackermann_steering_controller)
+
+find_package(catkin REQUIRED COMPONENTS
+  angles
+  control_toolbox
+  controller_interface
+  diff_drive_controller
+  gazebo_ros_control
+  hardware_interface
+  joint_limits_interface
+  nav_msgs
+  realtime_tools
+  roscpp
+  tf
+  urdf
+)
+
+find_package(gazebo REQUIRED)
+
+catkin_package(
+  INCLUDE_DIRS include
+  LIBRARIES ${PROJECT_NAME}
+  CATKIN_DEPENDS
```

Comment:
this is not complete, e.g. diff_drive_controller is missing.

Reply:
Fixed at https://github.com/ros-controls/ros_controllers/pull/356/commits/defd9a17a40d824a2818d808e898e3131badd466

Reply:
Fixed at https://github.com/ros-controls/ros_controllers/commit/a8087f71bb633822afe5b3611ec31eced4c17a29